File: fase06/ex01/ateZero.js
function ateZero (num) {
var z = []
    if (num === 0)
        return [0]
    else if (num>0)
        for (i=0;i<=num;i++) {
            z[i]=i
        }
    else
        for (j=0;j<=-num;j++) {
            z[j]=num+j
        }
    return z
}

File: python/poesiaVogon/fase06/ex01/ateZero.py
def ateZero (a):
    z=[]
    if a>0:
        for x in range(0,a+1):
            z.append(x)
        return z
    elif a<0:
        for x in range(a,1):
            z.append(x)
        return z
    else:
        return [0]
